nixos/modules/services/networking/tox-node.nix

```diff
@@ -11,8 +11,8 @@ let
     # fetchurl should be switched to getting this file from tox-node.src once
     # the dpkg directory is in a release
     src = pkgs.fetchurl {
-      url = "https://raw.githubusercontent.com/tox-rs/tox-node/master/dpkg/config.yml";
-      sha256 = "1431wzpzm786mcvyzk1rp7ar418n45dr75hdggxvlm7pkpam31xa";
+      url = "https://raw.githubusercontent.com/tox-rs/tox-node/5d972d0b692e411e16fdfcc46ac40379660e716e/dpkg/config.yml";
+      sha256 = "sha256-pUex0tYePy0P7tT5KceR3a1XdnVovDqWoFUsPeGaiw4=";
```

Comment:
Maybe we can get the file from `tox-node.src`? See the comment above.

Reply:
Updated, thanks.